src/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderDefineDefaultConstructor.cs

```diff
@@ -46,85 +46,85 @@ public TypeBuilder RetriveTestTypeBuilder(TypeAttributes typeAtt)
         }
 
         [Fact]
-        public void PosTest1()
+        public void TestDefaultConstructor1()
         {
             ExePosTest(TypeAttributes.Public | TypeAttributes.Class, MethodAttributes.Public, BindingFlags.Public | BindingFlags.Instance);
         }
 
         [Fact]
-        public void PosTest2()
+        public void TestDefaultConstructor2()
```

Comment:
Not a big deal, but should these be named based on their differences rather than based on a number?  Or should things like this move to become theories?  Or if we don't want to use a theory, just have a method that calls ExePosTest once for each of the combinations?